recipes/PyFstat/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "PyFstat" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5275804036a0d814acf111857afd46fbb3cba81e48f3d5275fbdd7219d75dddc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - matplotlib >=2.1
```

See https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib
```suggestion
    - matplotlib-base >=2.1
```